src/docs/Foundations/Typography/BodyStyles/index.mdx

```diff
@@ -1,27 +1,46 @@
 ---
 title: 'Body styles'
 order: 4
+description: 'A body style is a paragraphs or multiline texts of a pattern or a page. It must provide basic typographic styles like font-size, line-height and font-weight.'
```

Comment:
A body is a paragraph or multiline texts of a pattern or a page. It must provide basic typographic properties like font-size, line-height and font-weight.